projects/core/src/styles/shim.clr-ui.scss

```diff
@@ -908,3 +908,9 @@
     --clr-btn-icon-disabled-color: var(--cds-global-typography-color-100);
   }
 }
+
+[cds-theme~='light'] {
+  --clr-vertical-nav-bg-color: var(--cds-global-color-gray-200);
```

Comment:
There are alias like 'cds-alias-object-border-color' that should be used instead of directly taking the global color

Reply:
There are not any aliases for the NG equivalent grays. The aliases for interaction, object, container etc. in Core use construction globals which are cool gray - the NG grays are flat. We may wish to create aliases for these in the future if a 'classic Clarity NG' type of theme is needed.